### PR TITLE
f-spinner@0.1.0 - Create spinner component

### PR DESCRIPTION
**Pull request by FayeCarter**
> Component generated with the `fozzie-component` generator

`f-spinner`
### Added
- Initial version of spinner component

`fozzie-components`
### Added
- `f-spinner` to Circle CI cache.

### CODE REVIEW

File: packages/components/atoms/f-spinner/package.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    "fozzie"
+  ],
+  "scripts": {
+    "prepublishOnly": "yarn lint && yarn test && yarn build",
+    "build": "vue-cli-service build --target lib --name f-spinner ./src/index.js",
+    "lint": "vue-cli-service lint",
+    "lint:fix": "yarn lint --fix",
+    "lint:style": "vue-cli-service lint:style",
+    "test": "vue-cli-service test:unit",
+    "test-component:chrome": "cross-env-shell JE_ENV=local TEST_TYPE=component wdio ../../../../wdio-chrome.conf.js",
```
Comment (ashleynolan) [inline packages/components/atoms/f-spinner/package.json]: I would think you won't need component tests for this component? (may be wrong, but it seems like it'll be pretty simple?)

Comment (FayeCarter) [reply packages/components/atoms/f-spinner/package.json]: Great news, thanks! They're doing nothing but break 😄

**Review by BanksySan — DISMISSED** [review]

File: packages/components/atoms/f-spinner/src/components/Spinner.vue [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        :class="$style['c-spinner']"
+        data-test-id="spinner">
+        I am a VSpinner Component (GB)
+    </div>
+</template>
+
+<script>
+
+export default {
+    name: 'VSpinner',
```
Comment (minrwhite) [inline packages/components/atoms/f-spinner/src/components/Spinner.vue:12]: Can remove this, but happy if this is done in the follow up PR.

**Review by minrwhite — DISMISSED** [review]
One minor comment, but happy to approve - do you have any screenshots/videos of how this is meant look after completion?

File: packages/components/atoms/f-spinner/src/components/Spinner.vue (hunk 2 above)
Comment (FayeCarter) [reply packages/components/atoms/f-spinner/src/components/Spinner.vue:12]: 👍  I'll add to the next one

**Comment by FayeCarter** [comment]
> One minor comment, but happy to approve - do you have any screenshots/videos of how this is meant look after completion?

Thank you! 
I can add one to this PR https://github.com/justeat/fozzie-components/pull/1398. This is all just the generated stuff so not a lot to take a screenshot of at the moment

**Review by minrwhite — APPROVED** [review]

**Review by BanksySan — APPROVED** [review]

**Pull request merged**